docs/technical_reference/flowsheets/seawater_RO_desalination.rst

```diff
@@ -0,0 +1,200 @@
+Seawater RO Desalination
+========================
+
+Introduction
+------------
+
+This flowsheet represents a full-scale seawater reverse osmosis treatment facility.
+The flowsheet includes pretreatment, desalination, post-treatment, and waste handling unit processes available in WaterTAP.
+Unit models used on this flowsheet span the spectrum from simple to detailed.
+This flowsheet includes examples of several different types of modeling features available in WaterTAP, summarized in Table 1:
+
+.. csv-table::
+   :header: "Modeling Component", "Documentation"
+   
+   "Zero order unit and costing models", ":doc:`/technical_reference/unit_models/zero_order_unit_models/index`"
+   "Zero order costing package", ":doc:`/technical_reference/costing/zero_order_costing`"
+   "Zero order property package", ""
+   "Seawater property package", ":doc:`/technical_reference/property_models/seawater`"
+   "Reverse osmosis model", ":doc:`/technical_reference/unit_models/reverse_osmosis_0D`"
+   "Pump model", ""
+   "Energy recovery device model", ""
+   "Pressure exchanger model", ":doc:`/technical_reference/unit_models/pressure_exchanger`"
+   "WaterTAP costing package", ":doc:`/technical_reference/costing/watertap_costing`"
+   "Unit model costing packages", ":doc:`/technical_reference/costing/detailed_unit_model_costing`"
+   "WaterTAP database", ""
+   "IDAES Translator blocks", :doc:`idaes:reference_guides/model_libraries/generic/unit_models/translator`
+   "IDAES Product blocks", :doc:`idaes:reference_guides/model_libraries/generic/unit_models/product`
+   "IDAES Separator blocks", :doc:`idaes:reference_guides/model_libraries/generic/unit_models/separator`
+   "IDAES Mixer blocks", :doc:`idaes:reference_guides/model_libraries/generic/unit_models/mixer`
+
+The influent conditions are defined from the case study this flowsheet is modeled after. The relevant information is in Table 2:
+
+.. csv-table::
+   :header: "Description", "Value", "Units"
+
```

Comment:
I'd combine this table with the one below and place it in a "Flowsheet Specifications" section to better align with other flowsheet documentation. The sub-heading for this part of the combined table could just be "Influent"

Reply:
There should also be a section for "Degrees of Freedom" directly above the "Flowsheet Specifications" section.